src/components/atoms/ImageComponent/Image.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+import './Image.scss';
+
+
```

Comment:
add TODO comment here that we should handle the image loading with placeholder in future as a feature, something like

`// TODO Add loading on image onLoad`